etc/testing/start_postgres.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if ! docker ps | grep -q postgres
+then
+    echo "starting postgres..."
+    docker run -d \
+    -e POSTGRES_USER=pachyderm \
+    -e POSTGRES_DB=pgc \
+    -e POSTGRES_HOST_AUTH_METHOD=trust \
+    -p 32228:5432 \
+    postgres:latest
```

Comment:
Using the latest might get us in trouble at some point. Maybe we should be explicit about a version.

Reply:
They just released 13. Want to commit to that?